source/server/http/admin.h

```diff
@@ -53,12 +53,13 @@ class AdminImpl : public Admin,
   AdminImpl(const std::string& access_log_path, const std::string& profiler_path,
             const std::string& address_out_path, Network::Address::InstanceConstSharedPtr address,
             Server::Instance& server, Stats::ScopePtr&& listener_scope);
+  AdminImpl(const std::string& access_log_path, const std::string& profile_path,
+            Server::Instance& server);
```

Comment:
should we consider only having one AdminImpl ctor and using a setter for adding the address?

Reply:
Good idea; I will go do that.

Reply:
Done.